src/spec_procs.cpp

```diff
@@ -5545,7 +5592,10 @@ SPECIAL(chargen_unpractice_skill)
     }
 
     // Success. Lower the skill by one point.
-    GET_SKILL_POINTS(ch)++;
+    if (skills[skill_num].requires_resonance)
+      GET_CHANNEL_POINTS(ch)++;
```

Comment:
Logically, you probably want to only increase their channel points up to their attribute max, then put any extra points back into the general skill points pool. Otherwise you could have a situation where someone spends chanp + skillp on channels, then tries to untrain to pick up a different skill only to be locked into channeling and forced to remake.